.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "external/vcpkg"]
-	path = external/vcpkg
-	url = https://github.com/Microsoft/vcpkg.git
```

Comment:
why delete submodule ?

Reply:
I told that's because I couldn't get it to work with VS2022, and the project can't be generated through cmake-gui because it would pick up some of the cmakefiles in it and fail.

This is why I made a separate commit and told you to look for a better solution since I really have no idea about it.